pyproject.toml

```diff
@@ -27,55 +27,44 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "Typing :: Typed",
 ]
-dynamic = ["version"]
-dependencies = [
-    "loguru >= 0.7.0",
-]
 
-[project.optional-dependencies]
-# Use `constraints.txt`
-tests = [
-  "coverage",
-  "pytest",
-  "pytest-cov",
-]
-docs = [
-  "furo",
-  "myst-parser",
-  "Sphinx",
-  "sphinxcontrib-spelling",
-]
-stubs = [
-  # Typing stubs go here
-]
-typing = [
-  "crustypy-instance[stubs]",
-  "mypy",
-  "mypy-extensions",
-  "nox",
-  "pytest",
-]
-dev = [
-  "crustypy-instance[tests,docs,stubs,typing]",
-  "black",
-  "build",
-  "nox",
-  "pre-commit",
-  "python-semantic-release",
-  "ruff",
-  "semver",
-  "toml",
-]
+[tool.poetry.dependencies]
+python = "^3.8"
+loguru = "^0.7.0"
+
+[tool.poetry.group.tests.dependencies]
+coverage = "^7.2.7"
```

Comment:
`pytest-cov` comes with `coverage` bundled

Reply:
Isn't it better to depend on it directly, if I'm going to be using it by itself too?
I don't think I have yet, but I will be using it directly on Nox I think, for things like merging coverage reports and what not.

Reply:
you might want `coverage[toml]` then